readme.md

```diff
@@ -6,185 +6,165 @@
 
 ### Concepts
 
-- Compare and contrast CLIs / GUIs
-- Explain how command line usage can increase efficiency
-- Describe the anatomy of a command (statement, flags, arguments)
-- Explain what a path is, and why the 'current path' is important in the CLI
-- Explain the difference between an absolute path and a relative path
+- Compare and contrast CLIs with GUIs.
+- Explain how command line usage can increase efficiency.
+- Describe the anatomy of a command: statements, flags and arguments.
+- Explain what a path is and why the 'current path' is important in the CLI.
+- Explain the difference between absolute and relative paths.
 
 ### Mechanics
 
-- Setup your working directory / environment for WDI
-- List common commands to
-  - View the path of the current directory
-  - View the contents of a directory
-  - Navigate to different directories
-  - Manage files and directories
-- Open files and directories with Atom
-- List unsafe commands
+- Setup your working directory / environment for WDI.
+- List common commands to...
+  - View the path of the current directory.
+  - View the contents of a directory.
+  - Navigate to different directories.
+  - Manage files and directories.
+- Open files and directories with Atom.
+- List unsafe commands.
 
-## Framing(5/5)
+## Framing (5 minutes / 0:05)
 
-As developers, we need a tool to interface with our computers. OS's generally provide a graphical user interface or GUI so that we can interface with our computer. They also provide a UI that is strictly text based.
+As developers, we need a tool to communicate with our computers. Operating systems like Windows or OSX generally provide a graphical user interface (GUI) so that we can do this. They also, however, provide an interface that is strictly text-based: the Command Line (CLI).
 
-Turn & Talk: Given your exposure to the command line so far (prework and
-installfest), discuss the following questions:
+Turn & Talk: Given your exposure to the Command Line in the pre-work and Installfest, discuss and write down some answers for the following questions with a partner...
 
 * How is the CLI different from the GUI?
 * What do you like / dislike about using it?
-* It what ways might it be better or worse for Developers? (Compared to a GUI)
+* Compared to a GUI, in what ways might using the CLI be better or worse for developers?
 
+<!-- AM: What's this screenshot for? -->
```

Comment:
Gotcha! Then I think I'll add "debugging" as a benefit of CLI in the following section and move the screenshot there.